build-support/bin/check_pants_pex_abi.sh

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env bash
```

Comment:
+1 to python if we're keeping this, my thoughts were that this could be replaced with some form of integration testing in which we attempt to resolve and use the wheel with the specified abi, although this could maybe stay then as well.